sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/checks.sql

```diff
@@ -0,0 +1,34 @@
+#fail
+{{ is_unique(columns=["client_id"]) }}
+
+#fail
+ASSERT NOT EXISTS(
+  SELECT
+    daily.client_id
+  FROM
+    telemetry_derived.clients_daily_v6 daily
+  LEFT JOIN
+    telemetry_derived.clients_first_seen_v2 first_seen
+    USING (client_id)
+  WHERE
+    submission_date = @submission_date
+    AND first_seen.client_id IS NULL
+)
```

Comment:
```suggestion
);
```

Reply:
I thought something might be up with the new `generate-sql` task changes, but the semicolon should fix the task failure